test_infra/pyproject.toml

```diff
@@ -18,3 +18,4 @@ python = ">=3.6.2, <3.10"
 "aws-cdk.aws-rds" = "^1.115.0"
 "aws-cdk.aws-secretsmanager" = "^1.115.0"
 "aws-cdk.aws-ssm" = "^1.115.0"
+"aws-cdk.aws-opensearchservice" = "^1.124.0"
```

Comment:
I think all `cdk` packages should have the same minimum version.

Reply:
good point. made the change: [commit](https://github.com/AssafMentzer/aws-data-wrangler/commit/827c3bf4c79a71352a14b8260f4057e2b363c051)